ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -233,7 +234,7 @@ Filter By Resource Type And Check Output
     [Documentation]    Filter by resource type
     Select Checkbox Using Id    id=tutorial--check-box
     Verify The Resources Are Filtered
-    ...    expected_types=${EXPECTED_ITEM_RESOURCE_TYPE}    expected_number=14
+    ...    expected_types=${EXPECTED_ITEM_RESOURCE_TYPE}    expected_number=15
```

Comment:
Out of curiosity - do we know what item has been added here?

Reply:
I updated it to be a dynamic number instead of hardcoded now.